### PR TITLE
Updates in cogs, Split code in proper categories

### PR DESCRIPTION
**Pull request by janaSunrise**

### CODE REVIEW

**Comment by ItsDrike** [comment]
We need to wait for PR #1 to be merged and implement the changes later on so that the code will be `flake8` compatible. Also `requirements.py` doesn't need to contain all of the packages when using pipenv. `Pipenv` already contains them, we only need to include `pipenv` itself in the `requirements.py` and complete the installation with bare `Pipenv` using `pipenv sync`

**Comment by ItsDrike** [comment]
I noticed that you've added some cython files, that shouldn't be here at all. They should be removed

File: utils/wolframscrape.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import requests
+import setup
 
-APPID = "E75JHH-HAUP4AYJTT"
+APPID = setup.WOLFRAM_APPID
```
[the remark this answers is not in the document]
Comment (janaSunrise) [reply utils/wolframscrape.py:4]: We'll do it asap

File: cogs/study.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+from utils.wolframscrape import get_wolfram_data
+
+
+class Study(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.command()
+    async def pcalc(self, ctx):
+        pass
```
[the remark this answers is not in the document]
Comment (janaSunrise) [reply cogs/study.py:14]: ok, delete it then

File: cogs/support.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+class Support(commands.Cog):
+	def __init__(self, client):
+		self.client = client
+
+	@commands.command()
+	async def invite(self, ctx):
+		"""
+        Invite link for Bot
+        """
```
[the remark this answers is not in the document]
Comment (janaSunrise) [reply cogs/support.py]: hmm

**Pull request merged**